packages/hub/src/app/models/[owner]/[slug]/EditSquiggleSnippetModel.tsx

```diff
@@ -262,6 +264,87 @@ export const EditSquiggleSnippetModel: FC<Props> = ({
 
   const { height, ref } = useAvailableHeight();
 
+  const checkedVersion = useValidSquiggleVersion(version);
```

Comment:
Why does the error use a Toast, and not just an error message here? I would have expected that the component just show an error directly here instead.

Reply:
I'm not sure how that would work. Do you suggest I return a `result<...>` object from `useValidSquiggleVersion`, or maybe a `{ validVersion, error }` pair?

In that case, we'd have to make a decision on how to show an error in each case where we call `useValidSquiggleVersion`. That's just two places right now, but I expect the number will grow (for example, we'll also need it it on export view pages, after we implement versioned `SquiggleChart`). So it's possible but it's more work, and this problem won't happen too often.

Btw, I don't think that the error should be fatal, so throw in `useValidSquiggleVersion` that would rely on `ErrorBoundary` is worse; consider the scenario where we decide to prune some versions later. Keep only `0.8.0`, `0.9.0`, `0.10.0`, and round up other versions to the closest compatible version. So it's more of a warning thing than an error.

Reply:
Ah, if it's just a warning, then it's less important. If it were more of a fatal error, I'd probably prefer we find a way to generally implement it as a component, in many places if necessary.